test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SpecializationE2ETests.cs

```diff
@@ -294,17 +294,17 @@ public async Task StartAsync_SetsCorrectActiveHost_RefreshesLanguageWorkerOption
         public void ColdStart_JitFailuresTest(string fileName)
         {
             var path = Path.Combine(Path.GetDirectoryName(new Uri(typeof(HostWarmupMiddleware).Assembly.CodeBase).LocalPath), WarmUpConstants.PreJitFolderName, fileName);
-
             var file = new FileInfo(path);
 
             Assert.True(file.Exists, $"Expected PGO file '{file.FullName}' does not exist. The file was either renamed or deleted.");
 
-            JitTraceRuntime.Prepare(file, out int successfulPrepares, out int failedPrepares);
+            JitTraceRuntime.Prepare(file, out List<string> successfulPrepares, out List<string> failedPrepares);
+
+            var failurePercentage = (double) failedPrepares.Count / successfulPrepares.Count * 100;
 
-            var failurePercentage = (double) failedPrepares / successfulPrepares * 100;
-            
             // using 1% as approximate number of allowed failures before we need to regenrate a new PGO file.
-            Assert.True( failurePercentage < 1.0 , $"Number of failed PGOs are more than 1 percent! Current number of failures are {failedPrepares}. This will definitely impact cold start! Time to regenrate PGOs and update the {fileName} file!");
+            string failedSummary = string.Join(Environment.NewLine+Environment.NewLine, failedPrepares);
+            Assert.True(failurePercentage < 1.0 , $"Number of failed PGOs are more than 1 percent! Current number of failures are {failedPrepares}. This will definitely impact cold start! Time to regenrate PGOs and update the {fileName} file!\r\n{failedSummary}");
```

Comment:
This test is currently failing - I need to update the jittrace file

Reply:
Was it just windows that was failing or Linux too?

Reply:
I just updated this PR with the updated windows jittrace file and that test now passes. Just need the new linux file now

Reply:
Updated with new Linux jittrace as well